docs/en/siem/siem-ui.asciidoc

```diff
@@ -90,8 +106,8 @@ The map provides a visual overview of your network traffic. It is interactive,
 so you can start exploring data directly from the map. Hover over source and
 destination points to see more information, such as hostnames and IP addresses.
 To drill down, click a point and use the plus sign to add a field to the
```

Comment:
```suggestion
To drill down, click a point and use the filter icon to add a field to the
```

The plus sign has been replaced with the 'filter' icon used elsewhere in the app. It'll show up on hover over the field's value.

e.g.
<img width=200 src="https://user-images.githubusercontent.com/2946766/67533472-475ff680-f687-11e9-9644-6496a2c30103.png"/>

Reply:
Thanks